vizro-core/docs/pages/user-guides/extensions.md

```diff
@@ -88,3 +88,111 @@ For more information, see our documentation on [customizing CSS](custom-css.md)
 It is possible to create custom React.js components and add them directly to any Vizro dashboard so enabling you to code beneath both the Vizro and Dash layers and control React.js directly
 
 For more information, see the documentation on [using React.js components with Dash](https://dash.plotly.com/plugins)
+
+## Using custom functions in `yaml`/`json` configuration
```

Comment:
I think section should go as a subsection of the `Vizro customizations` section.